vms/platformvm/state/state.go

```diff
@@ -1470,7 +1470,11 @@ func (s *state) loadCurrentValidators() error {
 			return err
 		}
 
-		staker, err := NewCurrentStaker(txID, stakerTx, metadata.PotentialReward)
+		staker, err := NewCurrentStaker(
+			txID,
+			stakerTx,
+			stakerTx.(txs.ScheduledStaker).StartTime(),
```

Comment:
Can we properly check for the type cast here (and below)? I know this should never be able to fail... and that it will be removed in a followup PR once the startTime is persisted... But I'd prefer to keep the merged code "safe"

Reply:
Done